cmake/projects/lmdb/hunter.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# Copyright (c) 2017, David Hirvonen
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    lmdb
+    VERSION
+    0.9.21-p0
+    URL
+    "https://github.com/hunter-packages/lmdb/archive/v0.9.21-p0.tar.gz"
```

Comment:
I think this release created from the wrong branch:
```
CMake Error: The source directory "/.../9941f34/Build/lmdb/Source" does not
appear to contain CMakeLists.txt.
```

Use this release:
* https://github.com/hunter-packages/lmdb/releases/tag/v0.9.21-p1

Reply:
Done